File: DistanciaPuntos/DistanciaPuntos/Punto1.vb
Public Class Punto1
    Inherits Punto

    Public Sub New(x As Double, y As Double)
        Me.X = x
        Me.Y = y
    End Sub

    Public Function DistanciaPunto2(x As Double, y As Double)
        Return Math.Sqrt(((Math.Pow(x - Me.X, 2)) + (Math.Pow(y - Me.Y, 2))))
    End Function
End Class

File: DistanciaPuntos/DistanciaPuntosCiSharp/Punto1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DistanciaPuntosCiSharp
{
    class Punto1 : Punto
    {
        public Punto1(double x,double y)
        {
            this.X = x;
            this.Y = y;
        }

        public double DistanciaPunto2(double x,double y)
        {
            return Math.Sqrt(((Math.Pow(x - this.X, 2)) + (Math.Pow(y - this.Y, 2))));
        }
    }
}
